__mocks__/got.js

```diff
@@ -4,58 +4,42 @@ const species = require('./../assets/species.json');
 const planets = require('./../assets/planets.json');
 
 
-const got = (url, options) => {
-    let data, isError;
+const got = async (url, options) => {
+    let data;
     if (/^https:\/\/swapi.co\/api\/*/.test(url)) {
 
         const path = url.replace('https://swapi.co/api/', '').split('/');
-        const pathNum = parseInt(path[1])-1;
+        const object_id = parseInt(path[1]);
```

Comment:
Why did you suddenly switch from `camelCase` to `snake_case` for you variable naming?